uber/templates/regform.html

```diff
@@ -89,7 +91,7 @@
 
 {% if c.PAGE == 'confirm' %}
     <div class="form-group">
-        <label for="email" class="col-sm-2 control-label">Legal Name</label>
+        <label for="full_name" class="col-sm-2 control-label">Legal Name</label>
```

Comment:
`for=""` is an accessibility feature, so... we should definitely try to include it, yes.